utils/envvar.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+func IsEnvVarNonEmpty(env string) (string, error) {
```

Comment:
The distinction between unset/empty is not worth the complexity over just using `os.Getenv` and checking against `==`. `LookupEnv` is more for cases where the empty value is valid but still distinct from unset